bin/compile

```diff
@@ -336,7 +502,14 @@ if [[ -n "$COMPOSER_GITHUB_OAUTH_TOKEN" ]]; then
 		notice_inline 'Using $COMPOSER_GITHUB_OAUTH_TOKEN for GitHub OAuth.'
 	else
 		mcount "failures.dependencies.auth.COMPOSER_GITHUB_OAUTH_TOKEN"
-		error 'Invalid $COMPOSER_GITHUB_OAUTH_TOKEN for GitHub OAuth!'
+		error <<-EOF
+			Invalid token for GitHub OAuth!
+			
+			The OAuth token set in the '\$COMPOSER_GITHUB_OAUTH_TOKEN'
+			config var is invalid. Please ensure that you obtain a valid
+			token from https://github.com/settings/tokens and set it on the
+			app using 'heroku config:set COMPOSER_GITHUB_OAUTH_TOKEN=...'.
```

Comment:
But if the token is invalid, the only course of action is to re-generate one anyway, right?

Reply:
True. If you get this a lot it might help. Generally when someone hits a problem on Heroku, if I can get them to reproduce the issue on their local machine, then they are happier with us. 

For example if they are using a "valid" token, but maybe they didn't copy the first or last character, or they mis-typed it or something. Running an external command would help them to know that it was truly their fault and not ours. 

Not critical, but maybe something to think of in the future.